packages/advanced-logic/src/extensions/payment-network/near/any-to-near.ts

```diff
@@ -21,28 +23,10 @@ export default class AnyToNearPaymentNetwork extends AnyToNativeTokenPaymentNetw
    * @param {string} address address to check
    * @returns {boolean} true if address is valid
    */
-  protected isValidAddress(address: string, networkName?: string): boolean {
-    switch (networkName) {
-      case 'aurora':
-        return this.isValidMainnetAddress(address);
-      case 'aurora-testnet':
-      case 'near-testnet':
-        return this.isValidTestnetAddress(address);
-      case undefined:
-        return this.isValidMainnetAddress(address) || this.isValidTestnetAddress(address);
-      default:
-        throw new UnsupportedNetworkError(networkName, this.supportedNetworks);
-    }
-  }
-
-  private isValidMainnetAddress(address: string): boolean {
+  protected isValidAddress(address: string): boolean {
     return this.isValidAddressForSymbolAndNetwork(address, 'NEAR', 'aurora');
```

Comment:
I don't understand why we get rid of the testnet addresses validation. FYI they have different validation rules compared to mainnet.

Reply:
Near testnet now has its separate class: https://github.com/RequestNetwork/requestNetwork/blob/e57b267a1ac0b14e51f44ff95fe90b465a280409/packages/advanced-logic/src/extensions/payment-network/near/any-to-near-testnet.ts#L17